use_cases/gretel.json

```diff
@@ -75,11 +43,29 @@
       },
       "gtmId": "use-case-redact-pii"
     },
+    {
+      "title": "Generate high-quality time series data",
+      "description": "Use DoppelGANger to create accurate and correlated time-series data.",
+      "imageName": "timeseries.png",
+      "modelType": "synthetics",
```

Comment:
should this be `timeseries_dgan`?

Reply:
Maybe? I'm still a bit confused about how we break these apart tbh. That's not an option in our Console `MODEL_TYPE` enum, which is what i've been going off of so far. Happy to update if you think it should be!

Reply:
i updated this